doc/content/resources/multiapps_chps/chp_5_multiapps.md

```diff
@@ -0,0 +1,212 @@
+      style=display: block;margin-left:auto;margin-right:auto;width:40%;
+      id= FullSolveMultiApp
+      caption=Example of FullSolveMultiApp.
+
+ [FullSolveMultiApp] shows an example of a FullSolveMultiApp where Griffin is the main application and SAM is the child application that will be solved throughout the entire time domain every time it is called. FullSolveMultiApp is generally used when we only need the end-time solution of the child application. In this example, the end-time SAM solution is used for the multiphysics coupling with a Griffin eigenvalue calculation.
+
+#### TransientMultiApp
+
+In the [`TransientMultiApp`](https://mooseframework.inl.gov/source/multiapps/TransientMultiApp.html) the child application will be solved at a single timestep, following the same timestep as the parent application.  The child applications must use a transient `Executioner`. TransientMultiApp is used for performing coupled simulations with the parent and child application when both physics are marching in time together.
+
+!media media/resources/transientmultiapp.png
+      style=display: block;margin-left:auto;margin-right:auto;width:37%;
+      id= Transientmultiapp
+      caption=Example of Transientmultiapp.
+
+ [Transientmultiapp] shows an example of a `TransientMultiApp` where the main application (Griffin) and the child application (SAM) progress together in time.
+ `TransientMultiApp` is simultaneously advancing the time-steps of the main and child applications by default. This means that no coupling fixed point iterations are done at any time-step. This approach is commonly referred to as loose coupling.
+
+ If a tight coupling approach is desired,  Picard [`fixed point iterations`](https://mooseframework.inl.gov/syntax/Executioner/FixedPointAlgorithms/index.html) may be used by specifying appropriate `Executioner` parameters for the parent application. Picard iterations are nonlinear iterations for tight coupling which repeat the simulation at the same time-step until a convergence criterion is satisfied.  Picard iterations are particularly important if the physics solved in the different apps have very strong nonlinear effects.  
+
+The following two examples describe the use of the `TransientMultiApp`. The definitions of execute_on and positions will be given in the optional parameters.
+
+!listing
+[MultiApps]
+  [dynamic_executable]
+    type = TransientMultiApp
+    execute_on = 'TIMESTEP_END'
+    input_files = 'pebble.i'
+  []
+[]
+
+!listing
+[MultiApps]
+ [sub]
+    type = TransientMultiApp
+    app_type = BisonApp
+    input_files = 'Bison_input.i'
+    positions = '0   0   0
+                 0.5 0.5 0
+                 0.9 0.9 0'
+  []
+[]
+
+The minimum time step over the main and all child applications will be used, by default, unless `sub_cycling` is enabled. If sub_cycling is enabled this will allow faster calculations, as the child applications will be able to perform multiple time steps per execution.
```

Comment:
```suggestion
The minimum time step over the main and all child applications will be used, by default, unless `sub_cycling` is enabled. If sub_cycling is enabled this will allow faster calculations, as the parent app will not be forced to use the smaller time step from the child app.
```